test/trento/domain/cluster/cluster_test.exs

```diff
@@ -116,25 +160,121 @@ defmodule Trento.ClusterTest do
       )
     end
 
-    test "should return an error if the cluster was not registered yet and a command from a non-DC is received" do
-      assert_error(
-        [],
+    test "should add a new host to the cluster and keep the details as empty when a non-DC node message is received" do
```

Comment:
Do we need this one? the rule is that if details do not change, we do not emit a DetailsUpdated event. This is a consequence of that domain rule, which is already formalized in tests, so I do not think we need to maintain this test.

Reply:
done